src/code/z_player_call.c

```diff
@@ -1,11 +1,60 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_player_call/func_80160A90.s")
+#define FLAGS                                                                                               \
+    (ACTOR_FLAG_1 | ACTOR_FLAG_8 | ACTOR_FLAG_10 | ACTOR_FLAG_20 | ACTOR_FLAG_200000 | ACTOR_FLAG_2000000 | \
+     ACTOR_FLAG_4000000 | ACTOR_FLAG_80000000)
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_player_call/func_80160AF8.s")
+ActorFunc sPlayerCallInitFunc;
+ActorFunc sPlayerCallDestroyFunc;
+ActorFunc sPlayerCallUpdateFunc;
+ActorFunc sPlayerCallDrawFunc;
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_player_call/func_80160B40.s")
+void PlayerCall_Init(Actor* thisx, GlobalContext* globalCtx);
+void PlayerCall_Destroy(Actor* thisx, GlobalContext* globalCtx);
+void PlayerCall_Update(Actor* thisx, GlobalContext* globalCtx);
+void PlayerCall_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_player_call/func_80160B80.s")
+const ActorInit Player_InitVars = {
+    ACTOR_PLAYER,
+    ACTORCAT_PLAYER,
+    FLAGS,
+    GAMEPLAY_KEEP,
+    sizeof(Player),
+    (ActorFunc)PlayerCall_Init,
+    (ActorFunc)PlayerCall_Destroy,
+    (ActorFunc)PlayerCall_Update,
+    (ActorFunc)PlayerCall_Draw,
+};
```

Comment:
So this is part of Player? It has Player's InitVars in it...

Reply:
Well, yes, it is part of the Player system

Reply:
But it has PlayerCall functions in it.

Reply:
This topic was discussed this morning on the Discord server
https://discord.com/channels/688807550715560050/945521329002672178/945712870694457365